feedback/templates/public/create-survey.html

```diff
@@ -51,14 +66,14 @@
 				<div class='row twelve columns'>
 					
 					<div class='two columns'>.</div>
-					<div class='eight columns'>
+					<div class='eight columns' id='answerType'>
```

Comment:
This is a `<div>` class that's going to be cloned a lot, isn't it? If so, shouldn't it be a class instead of an id? (Ids are usually unique.)